src/_includes/scripts.html

```diff
@@ -59,10 +59,18 @@
   var oLeftNav = document.getElementById("left-navigation");
   var oMDContent = document.getElementById("markdown-body");
   var oMDToc = document.getElementById("markdown-toc");
+  var vw = Math.max(document.documentElement.clientWidth || 0, window.innerWidth || 0)
```

Comment:
Can we follow a typical js convention where the first letter dictates the object type (o = object, s = string, i = integer), so maybe something like iViewWidth